app/scripts/templates/user_pages/item.html

```diff
@@ -1,4 +1,4 @@
-<a class="image {{imagePosition}}" href="{{url}}">
+<a class="image {{imagePosition}}" href="{{_getUrl}}">
```

Comment:
Yeah it's on purpose. It's a hack until we fix the data. I want it to look wrong so we'll fix it :smile: